batch/plugins/org.jboss.tools.batch.ui/src/org/jboss/tools/batch/ui/quickfixes/BatchQuickFixMessages.properties

```diff
@@ -0,0 +1,33 @@
+##################################################################################
+### Copyright (c) 2015 Red Hat, Inc.
+### Distributed under license by Red Hat, Inc. All rights reserved.
+### This program is made available under the terms of the
+### Eclipse Public License v1.0 which accompanies this distribution,
+### and is available at http://www.eclipse.org/legal/epl-v10.html
+###
+### Contributors:
+###     Red Hat, Inc. - initial API and implementation
+##################################################################################
+
+CREATE_BATCHLET=Create Batch Batchlet
+CREATE_DECIDER=Create Batch Decider
+CREATE_ITEM_READER=Create Batch Item Reader
+CREATE_ITEM_WRITER=Create Batch Item Writer
+CREATE_ITEM_PROCESSOR=Create Batch Item Processor
+CREATE_CHECKPOINT_ALGORYTHM=Create Batch Checkpoint Algorithm
+CREATE_PARTITION_MAPPER=Create Batch Partition Mapper
+CREATE_PARTITION_REDUCER=Create Batch Partition Reducer
+CREATE_PARTITION_COLLECTOR=Create Batch Partition Collector
+CREATE_PARTITION_ANALYZER=Create Batch Partition Analyzer
+CREATE_JOB_LISTENER=Create Batch Job Listener
+CREATE_STEP_LISTENER=Create Batch Step Listener
+CREATE_CHUNK_LISTENER=Create Batch Chunk Listener
+CREATE_ITEM_READ_LISTENER=Create Batch Item Read Listener
+CREATE_ITEM_PROCESS_LISTENER=Create Batch Item Process Listener
+CREATE_ITEM_WRITE_LISTENER=Create Batch Item Write Listener
+CREATE_SKIP_READ_LISTENER=Create Batch Skip Read Listener
+CREATE_SKIP_PROCESS_LISTENER=Create Batch Skip Process Listener
+CREATE_SKIP_WRITE_LISTENER=Create Batch Skip Write Listener
+CREATE_RETRY_READ_LISTENER=Create Batch Retry Read Listener
+CREATE_RETRY_PROCESS_LISTENER=Create Batch Retry Process Listener
+CREATE_RETRY_WRITE_LISTENER=Create Batch Retry Write Listener
```

Comment:
Do we really need to add Batch to the every message? I mean Create Batch Batchlet vs. Create Batchlet.